packages/mock-repo/src/non-a11y-component/non-a11y-component.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+
+export class NonA11yComponent extends React.Component<{}> {
+
+  public render() {
+    return (
+      <button id="saveChanges" role="NOT AN ARIA ROLE">
```

Comment:
I think the id and the text content are unnecessary. 
```suggestion
      <button role="NOT AN ARIA ROLE" />
```